integrations/artifactory_buildinfo.rst

```diff
@@ -0,0 +1,63 @@
+.. _integrations_artifactory_build_info:
+
+Artifactory Build Info
+======================
+
+The `Artifactory build info <https://www.buildinfo.org/>`_ Artifactory build info is a recollection of the metadata of a build.
+This json-formatted file includes all the details about the build broken down into segments like version history, artifacts, project
+modules, dependencies, and everything that was required to create the build.
+
+Build infos are identified with a ``build name`` and a ``build number``, similar to how many CI services identify the builds.
+They are conveninently stored in Artifactory to keep track of the build metadata and to perform different operations like merging
+the metadata with other build infos or build promotion between repositories.
+
+Conan has support for the build info at the `extesions repository <https://github.com/conan-io/conan-extensions>` using the feature
+:ref:`custom commands<reference_commands_custom_commands>`, that provides additional commands to create and manage the build info files
+uploaded to Artifactory.
+
+How to install the build info custom commands
+---------------------------------------------
+
+Using the dedicated repository for Conan extensions https://github.com/conan-io/conan-extensions, it is as easy as:
+
+.. code-block: bash
+
+    $ conan config install https://github.com/conan-io/conan-extensions
+
+
+Generating a build info
+-----------------------
+
+A build info can be generating from a create or install command:
+
+.. code-block: bash
+
+    $ conan create . --format json -s build_type=Release > create_release.json
+
+Then upload the created package to your repository:
+
+.. code-block: bash
+
+    $ conan upload ... -c -r ...
+
+Now, using the JSON output form the create/install commands, a build info file can be generated:
+
+.. code-block: bash
+
+    $ conan art:build-info create create_release.json mybuildname_release 1 <repo> --server my_artifactory --with-dependencies > mybuildname_release.json
+
+And then uploaded to Artifactory:
+
+.. code-block: bash
+
+    $ conan art:build-info upload mybuildname_aggregated.json --server my_artifactory
+
+For more reference, see the full example at https://github.com/conan-io/conan-extensions/tree/main/extensions/commands/art#how-to-manage-build-infos-in-artifactory
+
+Managing the build infos
```

Comment:
I think this section fits more into a `.. seealso::` like in other parts of the docs that serve to provide additional references for the section.